### DISCUSSION TITLE
When Trino or iceberg \ hive s3 version?

### DISCUSSION THREAD

**Discussion by YuriyGavrilov** [apache/seatunnel#7616] [Ideas]
Just wish to have Trino source sink availability or\and iceberg - hive metastore s3 capabilities. Currently don't know how to use Seatunnel better with Trino instead of writing simple parquet files on s3.

**Comment by YuriyGavrilov** [comment]
For Trino maybe something will be here https://github.com/apache/seatunnel/issues/7695

**Reply by maxpiter** [reply]
Вам нужно синкать не в трино, а в каталог который саппортит iceberg (hive glue nessie). а триной будете подключаться к каталогу и s3 и читать таблицы.

**Reply by YuriyGavrilov** [reply]
@maxpiter да я так и сделал. Но там своя специфика еще. Если комит не закроется в каталог то файлы будут а данные в айсбер не появятся.

**Reply by maxpiter** [reply]
Да, прочитал Вашу статью. Когда seatunnel падает файлы видимо подвисают в промежуточном статусе, чтото типа коммита не проходит. Соответственно трино их не отображает потомму что в манифестах нет соотв записей.
